src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(new Random().nextInt(100));

```

Comment:
100 is the magic number.
Avoid using of magic number, wrap it into the Constant with proper name. 
```suggestion
        ball.setNumber(new Random().nextInt(SHOULD_BE_CONSTANT));

```